VueJS/components/List.js

```diff
@@ -1 +1,23 @@
-export default Vue.component('list', {})
+export default Vue.component('todo-list', {
+    props:['todos'],
+    data: function() {
+        return {
+            msgTmp: ''
+        }
+    },
+    template: '<ol>' +
+        '<li is="todo-item" ' +
+        'v-for="(todo, index) in todos" ' +
+        'v-bind:key="todo.id" ' +
+        'v-bind:todo="todo" ' +
+        'v-on:deleteTodo="todos.splice(index, 1)"' +
+        'v-on:editTodo="(editedTodo) => editTodo(editedTodo, todo)"' +
+        'todo.message="msgTmp"' +
```

Comment:
What does that do? I can't see a vue directive here.. :thinking:  I believe you can remove that line.